third_party/move/move-compiler-v2/src/logging.rs

```diff
@@ -0,0 +1,96 @@
+// Copyright © Aptos Foundation
+// Parts of the project are originally copyright © Meta Platforms, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+//! Configures the logging backend for the compiler and related code which uses
+//! the `log` crate (macros `info!`, `error!`, etc.).
+//!
+//! # How to Run the Logger
+//!
+//! By default logging is turned off. The environment variable `MVC_LOG` is
+//! used to control logging. Usages:
+//!
+//! ```
+//!     MVC_LOG = ""                # logs everything to console
+//!     MVC_LOG = "info"            # logs only info or higher to console
+//!     MVC_LOG = "<module_path_prefix>=info" # logs only info+ in matching modules
```

Comment:
Just a few lines below, I point to the full documentation of this format in `LoggerSpecification`. Can you check out these docs? There is no need for me to repeat this here.

Reply:
Sorry, I had misunderstood you to mean "Move module" since I misunderstood in our meeting yesterday that it would filter messages based on Move code module.  I had doubted that a standard library would include that feature, so the library doc didn't seem relevant.